drivers/flash/spi_nor.h

```diff
@@ -62,6 +63,7 @@
 
 /* Flash octal opcodes */
 #define SPI_NOR_OCMD_SE         0x21DE  /* Octal Sector erase */
+#define SPI_NOR_OCMD_BE         0xDC23  /* Octal Block erase */
```

Comment:
Can you provide some reference documentation for the command? I have been looking at ATXP032, from Adesto, which is OctaSPI device and they use the same opcodes as the other modes.

Reply:
Macronix MX25LM51245G on the STM b_u585i_iot02a_ns board. https://www.macronix.com/Lists/Datasheet/Attachments/8729/MX25LM51245G,%203V,%20512Mb,%20v1.1.pdf

Reply:
The SPI NOR command set is de-facto standard but it varies across devices.
I do not think that this is common command across SPI NOR. If this command does not appear in other OctalFlash spec sheets, than this is not a place for it here.
I was not able, actually, to confirm h21de to be common either.